### ISSUE TITLE
Data storage not visible on the Anycubic Kobra 2 Pro 3D printer

### ISSUE THREAD

**Issue by NoWo2000** [mrfenyx/RPi-Zero-W-WiFi-USB#1]
Hi @mrfenyx,
I found the repo and was very pleased, as I currently have the same use case with my Anycubic kobra 2 pro. 

Unfortunately I can't see any data on the printer. 
I have followed the automatic installation and have also checked all entries manually and everything works so far. I have set up the network share and can also access it from my laptop and save files there, but no data is displayed on the printer itself or the Rasberry Pi 4 I am using is not displayed at all.
Even after modifying the 5v pin and trying to use other cables. 
Do you have any ideas?
<img width="886" alt="SCR-20231204-qecn" src="https://github.com/mrfenyx/RPi-Zero-W-WiFi-USB/assets/56127795/a183862b-922d-46c5-bb24-9cf642edc91b">

**Comment by mrfenyx** [comment]
Hi, so to make sure I understand your situation. You used the script and set up and checked manually everything. You see the network share but when you connect the Raspberry Pi Zero W with a USB cable to the printer or to a other PC, you don't see anything on the drive.
Does the Pi actually get mounted on the PC? Does it appear as a USB drive?
For the printer, are there any other USB drives connected? If yes, disconnect them and only have the raspberry pi zero w connected.

**Comment by NoWo2000** [comment]
I don't use a Pi Zero 3, but a classic Rasberry Pi 4 with 4GB RAM, but that shouldn't be a problem. 
And to your assumption, yes I started the automatic script...
```bash
cd RPi-Zero-W-WiFi-USB
sudo chmod +x install.sh
sudo ./install.sh
```

... and then isolated the 5V pin. 


Even when connecting the Pi to my laptop, no data carrier is displayed. 
No other USB devices are connected to the printer itself.

**Comment by mrfenyx** [comment]
I don't think that the raspberry pi 4 or any other raspberry pis except the zero w and the zero 2 w can work as a USB drive unfortunately

**Comment by NoWo2000** [comment]
I think it's the USB gadget mode, which the Pi 4 doesn't have. Other USB ports do not work either. I can get hold of a Pi Zero W until Wednesday, then I'll try the auto setup again. 
Is there anything else that needs to be done after running the Skritpe and adjusting the USB cable? Or should the printer already recognise the data storage in addition to the SMB share? 

Thank you for your quick response.

**Comment by mrfenyx** [comment]
Ok. I will try it out too sometime. I suggest getting the Zero 2 W and installing Raspberry pi is lite 64 bit. The performance is way better!
